<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jon's Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sevenButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eightButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nineButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sixButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threeButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twoButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oneButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fourButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fiveButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="additionButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subtractionButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalsButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiplicationButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divisionButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decimalButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="firstLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Jonathan's</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>314</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculator</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>290</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>255</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="action_Logout">
   <property name="text">
    <string>Close &amp;File</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Logout</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionWeight_Densities">
   <property name="text">
    <string>Weight Densities</string>
   </property>
  </action>
  <action name="actionWeight_Masses">
   <property name="text">
    <string>Weight Masses</string>
   </property>
  </action>
  <action name="actionTester_Properties">
   <property name="text">
    <string>Tester Properties</string>
   </property>
  </action>
  <action name="actionLocal_Gravity">
   <property name="text">
    <string>Local &amp;Gravities</string>
   </property>
  </action>
  <action name="actionHygrometer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Hygrometer</string>
   </property>
  </action>
  <action name="actionEdit_Defualts">
   <property name="text">
    <string>Edit Defualts</string>
   </property>
  </action>
  <action name="actionEnable_Defaults">
   <property name="text">
    <string>Enable Defaults</string>
   </property>
  </action>
  <action name="actionDisable_Defaults">
   <property name="text">
    <string>Disable Defaults</string>
   </property>
  </action>
  <action name="actionGravity1">
   <property name="text">
    <string>Gravity1</string>
   </property>
  </action>
  <action name="actionGravity2">
   <property name="text">
    <string>Gravity2</string>
   </property>
  </action>
  <action name="actionGravity3">
   <property name="text">
    <string>Gravity3</string>
   </property>
  </action>
  <action name="actionGravity4">
   <property name="text">
    <string>Gravity4</string>
   </property>
  </action>
  <action name="actionPressure">
   <property name="text">
    <string>Pressure</string>
   </property>
  </action>
  <action name="actionMass">
   <property name="text">
    <string>Mass</string>
   </property>
  </action>
  <action name="actionPressure_2">
   <property name="text">
    <string>Pressure</string>
   </property>
  </action>
  <action name="actionMass_2">
   <property name="text">
    <string>Mass</string>
   </property>
  </action>
  <action name="actionDensity">
   <property name="text">
    <string>Density</string>
   </property>
  </action>
  <action name="actionForce">
   <property name="text">
    <string>Force</string>
   </property>
  </action>
  <action name="actionHumidity">
   <property name="text">
    <string>Humidity</string>
   </property>
  </action>
  <action name="actionTemperature">
   <property name="text">
    <string>Temperature</string>
   </property>
  </action>
  <action name="actionPower">
   <property name="text">
    <string>Power</string>
   </property>
  </action>
  <action name="actionGravity">
   <property name="text">
    <string>Gravity</string>
   </property>
  </action>
  <action name="actionArea">
   <property name="text">
    <string>Area</string>
   </property>
  </action>
  <action name="actionLength">
   <property name="text">
    <string>Length</string>
   </property>
  </action>
  <action name="actionEdit_Screen">
   <property name="text">
    <string>Edit Screen</string>
   </property>
  </action>
  <action name="actionEnvironment_Screen">
   <property name="text">
    <string>Environment Screen</string>
   </property>
  </action>
  <action name="actionCalc_Screen">
   <property name="text">
    <string>Calc Screen</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_IDWT">
   <property name="text">
    <string>&amp;IDWT</string>
   </property>
  </action>
  <action name="actionSet_Environment">
   <property name="text">
    <string>Set Environment</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <resources/>
 <connections/>
</ui>
